### DISCUSSION TITLE
Docker compose missing a /

### DISCUSSION THREAD

**Discussion by FlattusBlastus** [nextcloud/all-in-one#2214] [Questions] [answered]
https://github.com/nextcloud/all-in-one/blob/main/docker-compose.yml
      - /var/run/docker.sock:/var/run/docker.sock:ro #incorrect
Please change to:
      - //var/run/docker.sock:/var/run/docker.sock:ro

**Comment by Zoey2936** [comment]
it is correct

**Comment by Zoey2936** [comment]
what os do you run? what docker version? what docker-compose version?

**Reply by FlattusBlastus** [reply]
Windows Docker Desktop / WSL2

**Reply by Zoey2936** [reply]
I think you got your answer, the syntax is made for linux, in the same line is noted, that it needs to be changed on windows/rootless/macos

**Comment by szaimen** [comment, marked as answer]
Indeed for windows it needs to be changed by hand to a double-slash as noted in https://github.com/nextcloud/all-in-one#how-to-run-aio-on-windows. However for Linux this work correctly.
